### ISSUE TITLE
no backup is taken when docker exit

### ISSUE THREAD

**Issue by nnn-gif** [michielbdejong/snickers-proxy#41]
my server got restarted which resulted known running docker to exit, after restarting server there is no backup of database though i have backup of files

**Comment by michielbdejong** [comment]
good point! it will keep the binary data files, but not the export to .sql text file, it would be nice to try to do a mysqldump when the soft shutdown signal (I think it's called the HUP signal?) is received.
